src/components/OrderList/Orders.tsx

```diff
@@ -59,45 +59,42 @@ class Blotter extends React.Component<OrdersProps, TableSortState> {
         value: 'Asset pair',
         width: OrderCellWidth.Symbol
       },
-      {
-        sortDisabled: checkDataForSorting(this.state.data, 'id'),
-        className: 'right-align',
-        key: 'id',
-        value: 'OrderID',
-        width: OrderCellWidth.Id
-      },
       {
         sortDisabled: checkDataForSorting(this.state.data, 'side'),
         className: 'right-align',
         key: 'side',
         value: 'Side',
         width: OrderCellWidth.Side
       },
+      {
+        sortDisabled: checkDataForSorting(this.state.data, 'price'),
+        className: 'right-align',
+        key: 'price',
+        value: 'Price'
+      },
       {
         sortDisabled: checkDataForSorting(this.state.data, 'volume'),
         className: 'right-align',
-        key: 'volume',
-        value: 'Volume'
+        key: 'amount',
```

Comment:
does it work? it sorts by a _key_ prop and there is no amount prop in the `Order` or do i miss smth?